src/rsz/src/RepairSetup.cc

```diff
@@ -731,6 +751,165 @@ bool RepairSetup::removeDrvr(PathRef* drvr_path,
   return false;
 }
 
+// Estimate slack impact from driver removal.
+// Delay improvement from removed driver should be greater than
+// delay degradation from prev driver for driver input pin path.
+// Side input paths should absorb delay and slew degradation from prev driver.
+// Delay degradation for side input paths comes from two sources:
+// 1) delay degradation at prev driver due to increased load cap
+// 2) delay degradation at side out pin due to degraded slew from prev driver
+//
+//  prev_drv_pin ------>  (drvr_input_pin   drvr_pin)  ------>
+//               |
+//               ------>  (side_input_pin1  side_out_pin1) ----->
+//               |
+//               ------>  (side_input_pin2  side_out_pin2) ----->
+//
+bool RepairSetup::estimatedSlackOK(slackEstimatorParams* params)
```

Comment:
Take params by const& as it should never be null.

Reply:
Done